tests/test_s3.py

```diff
@@ -168,6 +176,12 @@ def assert_num_uploads_found(self, operation, num_uploads,
             num_uploads, amount_seen))
 
 
+class TestS3Client(BaseS3ClientTest):
+
+    def test_client_endpoint_is_patched(self):
+        self.assertIsInstance(self.client._endpoint, aiobotocore.endpoint.AioEndpoint)
```

Comment:
basically the `_get_endpoint` hook was removed and now we have to rewrite the whole `create_endpoint` method